examples/src/main/python/mllib/naive_bayes_example.py

```diff
@@ -38,8 +45,12 @@ def parseLine(line):
 
     sc = SparkContext(appName="PythonNaiveBayesExample")
 
+    WORK_DIR = './'
```

Comment:
removed.  i added this, because it's easy for me to modify the WORK_DIR.  agree, simplicity is more important for this sample code.